nbs/models.ipynb

```diff
@@ -57,6 +57,7 @@
     "import pandas as pd\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.            sigma = np.sqrt(1/(len(y)-1)*np.nansum(np.power(mod['residuals'],2)))
</pre><p>This line is a little bit hard to read. Maybe we can divide it, something like:</p><p> </p><pre class="ql-syntax" spellcheck="false">sigma = np.nansum(mod['residuals'] ** 2)
sigma = sigma / (len(y) - 1)
sigma = np.sqrt(sigma)
</pre><p> </p><p>Since these lines are repeated for each model, maybe we can create a new function, before <code>_quantiles</code>, to prevent errors:</p><p> </p><pre class="ql-syntax" spellcheck="false">def _calculate_sigma(residuals: np.ndarray):
    n = len(residuals) #residuals and y have the same length
    sigma = np.nansum(residuals ** 2)
    sigma = sigma / (n - 1)
    sigma = np.sqrt(sigma)
    return sigma
</pre><p> </p><p>And use it whenever necessary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>

Reply:
<p>This was implemented and used for the historic average, the naive, and the rwd. However, the seasonal naive uses </p><p> </p><p><code>sigma = sigma/(n-season_length)</code></p><p> </p><p>So in this case I computed sigma directly in the function. </p>

Reply:
<p>The <code>_calculate_sigma</code> function will be changed so that it can be used in all 4 simple methods. </p>